duracloud/duracloud
# Ensures CORS headers are configured properly in httpd proxy server.

**Ensures CORS headers are configured properly in httpd proxy server.**
* * *

**JIRA Ticket**: https://jira.lyrasis.org/browse/DURACLOUD-1306

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)

# What does this Pull Request do?
With the last release we upgraded to the latest beanstalk environment which uses a slightly different file structure for configuring the apache proxy server.   This update ensures that CORS and other httpd .conf files are being picked up by the beanstalk apache configuration process.
# How should this be tested?
Build and deploy (currently deployed to dev)

curl -u username:password https://<account>.<duracloud-host>/durastore/<space>/<resource> -i

Verify the presence of CORS headers in the response.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @duracloud/committers
